### PR TITLE
Update user-deletion-and-suppression guide.md

### PR DESCRIPTION
**Pull request by muhammadatifjav**
Update User Deletion and Suppression guide

### Proposed changes

There were some inconsistencies and missing details in the deletion and suppression guide, so I'm made some changes to mention Public API upfront, and then also add more details on what the UI shows and what Public API will show. 

### Merge timing
- ASAP once approved

### Related issues (optional)

https://segment.atlassian.net/browse/KCS-620

### CODE REVIEW

File: src/privacy/user-deletion-and-suppression.md [hunk 1]
```diff
@@ -14,72 +14,86 @@ In keeping with Segment's commitment to GDPR and CCPA readiness, Segment offers
...
-The deletion requests tab shows a log of all regulations with a deletion element along with status.
-
-Click a deletion to view its status across Segment and your connected destinations.
+The deletion requests tab shows a log of all regulations with a deletion element along with status. The deletion requests can take up to 30 days to process.
 
+In the Segment App (Settings > End User Privacy > Deletion Requests), you can click a userId to view its status in Segment internal systems, and in the connected destinations.
 
-## Programmatic User Deletion and Suppression using the API
+The deletion request can have one of the following statuses:
+1. "FAILED"
```
Comment (rchinn-segment) [inline src/privacy/user-deletion-and-suppression.md:88]:
```suggestion
1. `FAILED`
```

File: src/privacy/user-deletion-and-suppression.md [hunk 2]
```diff
@@ -14,72 +14,86 @@ In keeping with Segment's commitment to GDPR and CCPA readiness, Segment offers
...
+2. "FINISHED"
```
Comment (rchinn-segment) [inline src/privacy/user-deletion-and-suppression.md:89]:
```suggestion
2. `FINISHED`
```

File: src/privacy/user-deletion-and-suppression.md [hunk 3]
```diff
@@ -14,72 +14,86 @@ In keeping with Segment's commitment to GDPR and CCPA readiness, Segment offers
...
+3. "INITIALIZED"
```
Comment (rchinn-segment) [inline src/privacy/user-deletion-and-suppression.md:90]:
```suggestion
3. `INITIALIZED`
```

File: src/privacy/user-deletion-and-suppression.md [hunk 4]
```diff
@@ -14,72 +14,86 @@ In keeping with Segment's commitment to GDPR and CCPA readiness, Segment offers
...
+4. "INVALID"
```
Comment (rchinn-segment) [inline src/privacy/user-deletion-and-suppression.md:91]:
```suggestion
4. `INVALID`
```

File: src/privacy/user-deletion-and-suppression.md [hunk 5]
```diff
@@ -14,72 +14,86 @@ In keeping with Segment's commitment to GDPR and CCPA readiness, Segment offers
...
+5. "NOT_SUPPORTED"
```
Comment (rchinn-segment) [inline src/privacy/user-deletion-and-suppression.md:92]:
```suggestion
5. `NOT_SUPPORTED`
```

File: src/privacy/user-deletion-and-suppression.md [hunk 6]
```diff
@@ -14,72 +14,86 @@ In keeping with Segment's commitment to GDPR and CCPA readiness, Segment offers
...
+6. "PARTIAL_SUCCESS"
```
Comment (rchinn-segment) [inline src/privacy/user-deletion-and-suppression.md:93]:
```suggestion
6. `PARTIAL_SUCCESS`
```

File: src/privacy/user-deletion-and-suppression.md [hunk 7]
```diff
@@ -14,72 +14,86 @@ In keeping with Segment's commitment to GDPR and CCPA readiness, Segment offers
...
+7. "RUNNING"
```
Comment (rchinn-segment) [inline src/privacy/user-deletion-and-suppression.md:94]:
```suggestion
7. `RUNNING`
```

**Review by rchinn-segment — CHANGES_REQUESTED** [review]
Thanks for submitting @muhammadatifjav! A few minor suggestions to address, then I'll include this in our next deploy.

**Review by muhammadatifjav — COMMENTED** [review]
Accepted all suggestions ! Thanks for all corrections Ryan!

**Review by rchinn-segment — APPROVED** [review]

**Pull request merged**